object/token.go

```diff
@@ -124,6 +124,7 @@ func GetTokenByRefreshToken(refreshToken string) (*Token, error) {
 func GetTokenByTokenValue(tokenValue, tokenTypeHint string) (*Token, error) {
 	switch tokenTypeHint {
 	case "access_token":
+	case "access-token":
 		token, err := GetTokenByAccessToken(tokenValue)
```

Comment:
I am still installing the packages and trying to run the server mode of the casdoor app, but in the meantime, I checked the RFC for Oauth 2 and the values before for tokenTypeHint were correct. "access_token" and "refresh_token" are the values listed here:
https://datatracker.ietf.org/doc/html/rfc7009#section-2.1

Reply:
I know, but in casdoor the token_type is set to `-` instead of `_`, I will try to fix it in later PR, I must ensure the compatibility.